bin/send_metrics.py

```diff
@@ -1,57 +1,25 @@
 # pylint: skip-file
 # flake8: noqa
 
-"""
-Script that sends generic metrics messages to sentry locally
-
-
-Overview
-
-This script is designed to be used when creating a new use case ID for the first
-time for the generic metrics platform.
-
-
-Usage
```

Comment:
I would keep the usage section of the document to make it easier for users to use the script.

Reply:
Oops sorry, it had auto merge enabled, I will add it back in later!